### PR TITLE
Update to joshua language packs

### PR DESCRIPTION
**Pull request by lewismc**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by dunn** [comment]
SHA mismatch on one of them:
```
==> Downloading https://home.apache.org/~lewismc/language-pack-ru-en-2016-11-04.tar.gz
Warning: Resource ru-en-hiero-pack reports different SHA256: bf42ea5ecedb7cd992a44f95f73f1c26e73c91b8756edf4f8d50d1bbaaf9a351
Downloaded to: /Users/brew/Library/Caches/Homebrew/joshua--ru-en-hiero-pack-2016-11.tar.gz
SHA256: 9a1396ca804679ff27dfca34fada15edee81acb31892b380f39252051161f5fa

File: Formula/joshua.rb [hunk 1]
```diff
@@ -14,24 +14,24 @@ class Joshua < Formula
     sha256 "176fa47a6a2722fb5b6bf1e2efba8da32bab6355f3d844424a817882ed7b3a8e" => :mavericks
   end
 
-  option "with-es-en-phrase-pack", "Build with Spanish–English phrase-based model [1.9 GB]."
```
Comment (dunn) [inline Formula/joshua.rb]: Are these packs no longer available?

Comment (lewismc) [inline Formula/joshua.rb]: They are available however they are not compatible with the new version of Joshua (master).

**Comment by lewismc** [comment]
Thank you for review @dunn I appreciate it. I've uploaded a new tar.gz file and I am no longer getting the signature mismatch
```
lmcgibbn@LMC-056430 /usr/local/joshua_resources/russian_experiments $ shasum -a 256 language-pack-ru-en-2016-11-04.tar.gz
bf42ea5ecedb7cd992a44f95f73f1c26e73c91b8756edf4f8d50d1bbaaf9a351  language-pack-ru-en-2016-11-04.tar.gz
```
Thanks again for further review.

File: Formula/joshua.rb (hunk 1 above)
Comment (dunn) [inline Formula/joshua.rb]: OK, thanks.  Do you know if they will be compatible in the near future?  Homebrew doesn't currently handle removing options very well, so it'd be nice to keep them if possible.

**Comment by lewismc** [comment]
Hi @dunn OK I've retained the language packs, added a new build (to accommodate our new Maven build in Joshua) and also ensured that tests pass as do all executable scripts. The again for your review.

File: Formula/joshua.rb (hunk 1 above)
Comment (MikeMcQuaid) [inline Formula/joshua.rb]: Would be good to keep them if they work with stable and just not use them for `build.head?`

Comment (lewismc) [inline Formula/joshua.rb]: The new language packs I've update with are compatible with stable and HEAD. Thanks for checking out.

**Comment by MikeMcQuaid** [comment]
Can you fix the `brew style` failures (https://bot.brew.sh/job/Homebrew%20Core/11097/version=yosemite/testReport/junit/brew-test-bot/yosemite/style_joshua/) 

Installation is also failing: https://bot.brew.sh/job/Homebrew%20Core/11097/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install_joshua/

**Comment by lewismc** [comment]
@MikeMcQuaid ack will do. Thank you

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Closing now, please reopen when the problem is fixed.

**Comment by lewismc** [comment]
ack thank you @fxcoudert I will do

**Pull request closed without merging**